buildSrc/src/main/java/Dependencies.kt

```diff
@@ -31,6 +31,7 @@ object AndroidSdk {
 object Apollo {
     const val apolloRuntime = "com.apollographql.apollo3:apollo-runtime:${Versions.apollo}"
     const val apolloNormalizedCache = "com.apollographql.apollo3:apollo-normalized-cache:${Versions.apollo}"
+    const val apolloNormalizedCacheSqliteAndroid = "com.apollographql.apollo3:apollo-normalized-cache-sqlite-android-debug:${Versions.apollo}"
```

Comment:
I was somehow hoping that Gradle would resolve the appropriate artifact "automagically" (android vs others, debug vs release). Would it work if you used the plain `com.apollographql.apollo3:apollo-normalized-cache-sqlite` instead?

Reply:
Yep - fixed by John here https://github.com/joreilly/StarWars/pull/15/files#diff-75b31066e97e27961c3c9e9668c539a133d73c10398dc76552df3f0119810ac0R35